etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
```

Comment:
Yes, it is for all Tendrl components to understand if the feature is enabled or not. This is for testing and staging purpose fixes #13